src/foam/u2/view/PasswordView.js

```diff
@@ -18,7 +18,85 @@
 foam.CLASS({
   package: 'foam.u2.view',
   name: 'PasswordView',
-  extends: 'foam.u2.tag.Input',
+  extends: 'foam.u2.View',
 
-  properties: [ [ 'type', 'password' ] ]
+  css: `
+    .input-field-container {
```

Comment:
These are global names. Try not to pollute the global namespace, so use ^ and myClass() instead.